services/resources/resources.ts

```diff
@@ -0,0 +1,21 @@
+export const fetchResourcesData = async (idUser: string) => {
+  const userId = idUser;
+  console.log(userId, "id del usuario en recursos fetch");
```

Comment:
Same comment as above. Please keep the English format, even for console information. If the dev team has any other agreement, or I missed any meeting around this, please let me know to avoid the translation.